nvidia/cheminformatics/utils/fileio.py

```diff
@@ -31,6 +36,56 @@ def initialize_logfile(benchmark_file=BENCHMARK_FILE):
             fh.write(f'# {config_message}\n')
             fh.write('date,benchmark_type,step,time(hh:mm:ss.ms),n_molecules,n_workers,metric_name,metric_value\n')
 
+class MaticsLogger(object):
```

Comment:
I think this should be MetricsLogger, correct?

Reply:
> I think this should be MetricsLogger, correct?

Yes, indeed.